Form1.cs

```diff
@@ -37,8 +38,8 @@ public Form1()
 
         }
 
-        Button tank = new Button();
-
+       
+        List<Button> tanks = new List<Button>();
```

Comment:
Should be a different data structure, so as not to iterate on each received coordinate